cmds/core/backoff/main.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2012-2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Run a command, repeatedly, until it succeeds or we are out of time
```

Comment:
Make this a single line and put the rest in the description.

Reply:
done